apps/src/applab/designMode.js

```diff
@@ -250,6 +250,15 @@ designMode.updateProperty = function (element, name, value) {
       break;
     case 'icon-color':
       element.setAttribute('data-icon-color', value);
+      const imageUrl = element.getAttribute('data-canonical-image-url');
+      if (ICON_PREFIX_REGEX.test(imageUrl)) {
+        const url = assetPrefix.renderIconToString(imageUrl, element);
+        if (element.nodeName == "IMG") {
```

Comment:
@islemaster it appears that this line has broken the circleci staging build: https://circleci.com/gh/code-dot-org/code-dot-org/32151
```
[CDO]/apps/src/applab/designMode.js
  251:5   error  Unexpected lexical declaration in case block  no-case-declarations
  256:30  error  Expected '===' and instead saw '=='           eqeqeq
```